src/router.ts

```diff
@@ -29,6 +29,45 @@ router.get('/dash/image', async (req, res, next) => {
         next(e)
     }
 })
+router.get('/dash/imageproxy', async (req, res, next) => {
```

Comment:
Why not return the image blob from `/dash/image` directly? It already has the image url and it also lets you avoid making user-specified fetch requests server-side

Reply:
🤯 Great idea. What if you or the app sw managers gave that task to a new sw person so they can learn how cluck works for when you are gone?